process-engine/consumer_api_meta
# 📦 Change to eslint and tsc

## Changes

1. Perform builds with pure tsc and remove gulp entirely
2. Switch from TSLint to ESLint
3. Implement the new styleguide we use with eslint
4. Let linter errors cause the Jenkins Builds to fail

## Issues

PR: #81

## How to test the changes

- Running `npm run build` works a great deal faster than before
- Linting is now possible and works as expected
- The `prepare` step of the Jenkins builds will now also run a great deal faster:
    - Compare job durations [here](https://ci.process-engine.io/blue/organizations/jenkins/process-engine_node-lts%2Fconsumer_api_meta/activity)
